src/anomalib/utils/callbacks/metrics.py

```diff
@@ -58,7 +58,7 @@ def __init__(
 
     def setup(
         self,
-        trainer: "trainer.AnomalibTrainer",
+        trainer: "engine.Engine",
```

Comment:
Just to double check, is `trainer: "anomalib.Engine"` still giving circular import issues?